ledger/complete/wal/checkpointer.go

```diff
@@ -769,3 +773,43 @@ func readCheckpointV5(f *os.File) ([]*trie.MTrie, error) {
 
 	return tries, nil
 }
+
+// requestDropFromOSFileCache requests the specified file
+// be dropped from OS file cache.
+// CAUTION: Returns nil without doing anything if GOOS != linux.
```

Comment:
```suggestion
// The use case is that when a new checkpoint is generated, OS file cache will still cache the entire checkpoint file in memory, causing the memory usage to stay high. Dropping it could free up the memory.
// CAUTION: Returns nil without doing anything if GOOS != linux.
```